api/v1/fcc.go

```diff
@@ -7,26 +7,16 @@ type FccFrnResponse struct {
 
 // Filer struct
 type Filer struct {
-	Form499ID                  string                     `json:"form_499_id"`
-	FilerIDInfo                FilerIDInfo                `json:"filer_id_info"`
-	AgentForServiceOfProcess   AgentForServiceOfProcess   `json:"agent_for_service_of_process"`
-	FCCRegistrationInformation FCCRegistrationInformation `json:"fcc_registration_information"`
-	JurisdictionStates         []string                   `json:"jurisdiction_states"`
+	Form499ID   string      `json:"form_499_id"`
```

Comment:
i think we can use `filer_id` instead of `Form499ID` as we use `FilerID` internally everywhere

Reply:
good point